plugin/priv_cc.lua

```diff
@@ -0,0 +1,9 @@
+-- if you need to set a special name for privilege, use "beerchat.name_priv_cc = spec_name" in minetest.conf
+local name_priv2cc = minetest.settings:get("beerchat.name_priv_cc")
+if name_priv2cc == '' then name_priv2cc = 'chat_creator' end
+minetest.log("info", "to create channel add new priv: " .. name_priv2cc)
+local S = minetest.get_translator("name_priv2cc")
```

Comment:
seems this mod currently has no support for translation right now, but we probably should use the mod name here? at least iirc thats what is traditionally done.

Reply:
Translator thing could also be dropped for now and later done for everything at once:
* Old request #82

Reply:
is there some standard way to refer to a specific mod in the description?